Enhance DDEV and MCP (#3334)
ezsystems/developer-documentation: docs/ai/mcp/mcp_usage.md

!!! note "Storage choice recommendations"

    Filesystem storage is convenient for the sake of this example and for testing.

>     For production, it is recommended that you use Redis or Valkey to share cache among the cluster.

Reviewer: ```suggestion
    For production, it is recommended that you use Redis or Valkey to share cache among the cluster and improve performance.
```
I'm guessing here, but usually cache improves performance?

Author: @mnocon 
It's Redis cache instead of File storage cache. But I guess it also probably improve perf as well.

>     `discovery_cache: ~` could have been used for developement so cache pool wouldn't have to be cleared after each change.

Reviewer: ```suggestion
    For development, you can set `discovery_cache: ~` to avoid clearing the cache after each change.
```
To avoid `could have been` and `wouldn't have to be cleared`
`For development` mirrors the sentence above
`Avoid` or `skip` could be used, I'm not sure which one would be better here

>      But it's not recommended for production.

Reviewer: ```suggestion
```
IMHO the production recommendation is 2 sentences above, no need to repeat this.

Author: In this case, it's make sens to keep the performance argument.

>      By using the filesystem storage, this example can illustrate that cache pool needs to be cleared to have the latest capabilities as when deploying into production.

Reviewer: ```suggestion
     This example uses the filesystem storage to illustrate that you have to clear the cache pool to refresh the available capabilities, exactly as when deploying into production.
```
or "to see the new capability", or "to use the new capability"

Author: available is good as it can be not-new, just enabling/disabling capabilities.
